pkg/render/logstorage/esgateway/esgateway.go

```diff
@@ -46,9 +46,9 @@ const (
 	Port                  = 5554
 
 	ElasticsearchHTTPSEndpoint = "https://tigera-secure-es-http.tigera-elasticsearch.svc:9200"
-	ElasticsearchPort          = 9200
```

Comment:
Could you please put this file back the way it was and just leave it as duplicated? We're trying to avoid importing the render package into these subpackages, and duplication is acceptable to avoid it.

We're ultimately trying to put all the components in the render package in their own subpackages to clean things up.